app/diagnostics.go

```diff
@@ -440,6 +440,7 @@ func (a *App) trackConfig() {
 		"isdefault_login_button_color":        isDefault(*cfg.LdapSettings.LoginButtonColor, ""),
 		"isdefault_login_button_border_color": isDefault(*cfg.LdapSettings.LoginButtonBorderColor, ""),
 		"isdefault_login_button_text_color":   isDefault(*cfg.LdapSettings.LoginButtonTextColor, ""),
+		"isempty_group_filter":                isDefault(*cfg.LdapSettings.GroupFilter, ""),
```

Comment:
Propose we use `isdefault_group_filter` for consistency with other similar settings.

1/5